processing/src/main/java/io/druid/query/filter/DruidDoublePredicate.java

```diff
@@ -19,12 +19,36 @@
 
 package io.druid.query.filter;
 
-
+/**
+ * DoublePredicate is only supported in Java 8+, so use this to avoid boxing when a double predicate is needed.
```

Comment:
This comment is stale because now the Druid's interface has two methods.

Reply:
Please add "Note: this is not a `{@link PublicApi}` or an `{@link ExtensionPoint}` of Druid."

Reply:
done. removed stale comment and added note.